README.md

```diff
@@ -384,6 +384,11 @@ format:
   - OpenReview: 7, 7, 6
   - ExpEnv: None
 
+- [Deep Hierarchical Planning from Pixels](https://openreview.net/forum?id=wZk69kjy9_d)
+  - Danijar Hafner, Kuang-Huei Lee, Ian Fischer, Pieter Abbeel
+  - Key: hierarchical RL, long-horizon and sparse reward tasks
+  - OpenReview: 6,6,5
```

Comment:
For consistency of context, it would be better to separate these scores by spaces. The next two are similar.

Reply:
fixed.

Reply:
Done!